cli/azd/pkg/tools/ensure.go

```diff
@@ -5,8 +5,13 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"log"
 )
 
+// Stores tools that have already been verified to being installed
+// This will save us time having to re-check downstream in other components
+var confirmedTools map[string]bool = map[string]bool{}
```

Comment:
I think we use `map[string]struct{}` to represent a `Set` instead of using the bool.

Reply:
going to keep as is for now.